networking/network_observability/network-observability-overview.adoc

```diff
@@ -9,17 +9,11 @@ toc::[]
 Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The Network Observability Operator uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored network flows information in the {product-title} console for further insight and troubleshooting.
 
 [id="dependency-network-observability"]
-== Dependency of Network Observability Operator
-
-The Network Observability Operator requires the following Operators:
-
-* Loki: You must install Loki. Loki is the backend that is used to store all collected flows. It is recommended to install Loki by installing the Red Hat Loki Operator for the installation of Network Observability Operator.
-
-[id="optional-dependency-network-observability"]
 == Optional dependencies of the Network Observability Operator
 
+* Loki: Loki is the backend that is used to store all collected flows. It is recommended to install Loki to use with the Network Observability Operator. You can choose to use xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki], but there are some considerations for doing this, as described in the linked section. If you choose to install Loki, consider using the Red Hat Loki Operator, as it is supported by Red Hat.
 * Grafana: You can install Grafana for using custom dashboards and querying capabilities, by using the Grafana Operator. Red Hat does not support Grafana Operator.
-* Kafka: It provides scalability, resiliency and high availability in the {product-title} cluster. It is recommended to install Kafka using the AMQ Streams operator for large scale deployments.
+* Kafka: Kafka provides scalability, resiliency and high availability in the {product-title} cluster. It is recommended to install Kafka using the AMQ Streams operator for large scale deployments.
```

Comment:
I'm not a huge fan of repeating `Loki: Loki` / `Kafka: Kafka` but it may be really personnal. I know it's common to do this in the docs.

If you agree we could trick using the downstream names in front like `Loki Operator: Loki` / `AMQ Streams: Kafka`

Reply:
@jpinsonneau Should we also mention IPFIX here?